cargo/cargo_build_script_runner/lib.rs

```diff
@@ -50,11 +50,13 @@ impl BuildScriptOutput {
     fn new(line: &str) -> Option<BuildScriptOutput> {
         let split = line.splitn(2, '=').collect::<Vec<_>>();
         if split.len() <= 1 {
+            print!("{}", line);
             return None;
         }
         let param = split[1].trim().to_owned();
         let key_split = split[0].splitn(2, ':').collect::<Vec<_>>();
         if key_split.len() <= 1 || key_split[0] != "cargo" {
+            print!("{}", line);
```

Comment:
nit: move the line below the command

Reply:
I'm not quite sure what you mean here - what are you hoping will appear before this?